.buildkite/pipeline.yml

```diff
@@ -17,6 +17,15 @@ steps:
     agents:
       system: x86_64-linux
 
+  - label: 'deploy (staging) 🚀'
```

Comment:
`staging` is the new CI environment so we want to deploy changes to `master` there